src/components/pages/CallPage.jsx

```diff
@@ -49,14 +51,24 @@ export default class CallPage extends React.Component {
     render() {
         let lane = this.props.lane;
         let call = this.props.calls.getIn(['callList', 'items', lane.get('callId')]);
+        let callInstructions;
+        if (lane.get('step') === "call") {
+            callInstructions = <ViewSize render={({size}) => (
+                <CallInstructions size={size} call={call} lane={lane}/>
+            )}/>
+        }
 
         return (
             <div className="CallPage">
                 <LaneSwitch/>
-                <div className="CallPage-lanes">
-                    <CallLane key={ lane.get('id') }
-                        lane={ lane } call={ call }/>
-                </div>
+                {callInstructions}
+                <ViewSize render={({size}) => (
+                    <div className="CallPage-lanes">
+                        <CallLane key={ lane.get('id') }
+                            lane={ lane } call={ call }
+                            size={size}/>
```

Comment:
Inconsistent style for JSX attributes. You tend to not enclose attribute expressions in spaces (`{ myVar }`). Most of our code does. In this particular snippet, both styles are used in the same component instance. This is not a big deal, and not something that you need to change in this PR, but it's worth considering for future work (and we really need to start linting code soon).